### PR TITLE
[INJIMOB-903] fix iOS backup & restore issues

### PR DESCRIPTION
**Pull request by KiruthikaJeyashankar**
Issues resolved In iOS
1. , whenever we open the Backup & Restore screen,  fetch the last backup info from cloud
2. we wont face the Backup & Restore screen stuck in loading issue
3. MUlti device issue is resolved

### CODE REVIEW

File: shared/CloudBackupAndRestoreUtils.ts [hunk 1]
```diff
@@ -81,16 +81,44 @@ class Cloud {
...
+   * TODO: Remove the test call(get profile info) to reduce extra api calls
+   */
   static async getAccessToken() {
     try {
       const tokenResult = await GoogleSignin.getTokens();
+
+      try {
+        request(
+          'GET',
+          `https://www.googleapis.com/oauth2/v1/userinfo?alt=json&access_token=${tokenResult.accessToken}`,
```
Comment (vijay151096) [inline shared/CloudBackupAndRestoreUtils.ts:94]: api's can be specified in api.ts file

File: shared/fileStorage.ts [hunk 2]
```diff
@@ -82,38 +83,32 @@ export const getBackupFilePath = (
...
   const compressedFilePath = await compressFile(fileNameSansExtension);
   await removeFile(fileNameSansExtension);
-  const backupFileMeta = await new FileStorage().getInfo(compressedFilePath);
-  return backupFileMeta;
+  return await new FileStorage().getInfo(compressedFilePath);
+}
+
+export async function cleanupLocalBackups() {
+  const isDirectoryExists = await new FileStorage().exists(backupDirectoryPath);
+  console.log('patel', isDirectoryExists);
```
Comment (vijay151096) [inline shared/fileStorage.ts:96]: can we remove the unwanted loggers

File: shared/fileStorage.ts [hunk 3]
```diff
@@ -82,38 +83,32 @@ export const getBackupFilePath = (
...
+  if (isDirectoryExists) {
+    const availableBackupDirFiles =
+      await new FileStorage().getAllFilesInDirectory(backupDirectoryPath);
+    console.log('patel', availableBackupDirFiles);
```
Comment (vijay151096) [inline shared/fileStorage.ts:100]: can we remove the unwanted loggers

Comment (KiruthikaJeyashankar) [reply shared/fileStorage.ts]: Resolved

File: shared/fileStorage.ts (hunk 2 above)
Comment (KiruthikaJeyashankar) [reply shared/fileStorage.ts]: Removed

**Pull request merged**